src/flavors/__tests__/__snapshots__/openapi.test.ts.snap

```diff
@@ -134,6 +154,20 @@ Object {
             "nullable": true,
             "type": "number",
           },
+          "nullableObjectArrayValue": Object {
+            "items": Object {
```

Comment:
Notice that we are correctly producing JSON Schema types with `items`, both with a `$ref` (here) and a primitive (next).